Sources/Signal+SwiftConcurrency.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Signal+SwiftConcurrency.swift
+//  ReactiveSwift
+//
+//  Created by Marco Cancellieri on 2021-11-11.
+//  Copyright (c) 2021 GitHub. All rights reserved.
+//
+#if compiler(>=5.5) && canImport(_Concurrency)
+import Foundation
+
+@available(macOS 10.15, iOS 13, watchOS 6, tvOS 13, macCatalyst 13, *)
+extension Signal {
+	public var asyncThrowingStream: AsyncThrowingStream<Value, Swift.Error> {
```

Comment:
I think we should use `AsyncStream<Result<Value, Error>>` to maintain the error type.

Reply:
Bump